README.md

```diff
@@ -155,6 +155,9 @@ PyPI installable dependencies are listed in the [requirements.txt](https://githu
 
 `pip install -r requirements.txt`
 
+### Optional dependencies
+- [pillow](https://pypi.org/project/Pillow/) to open cover art image.
```

Comment:
This can be written like this:

```yaml
- [pillow](https://pypi.org/project/Pillow/), for supporting the cover art feature (to decode the image files)
```